src/components/ProfileHeader/ProfileHeader.module.css

```
@@ -0,0 +1,64 @@
+.ProfileHeader {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background-color: white;
+  border-radius: 4px;
+  border: 1px solid silver;
+  text-align: center;
+}
+
+.Avatar {
+  max-width: 200px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
+}
+
+.Text {
+  text-align: center;
+  align-self: stretch;
+}
+
+.Center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.FilledLike {
+  text-align: center;
+}
+
+.Comment {
+  text-align: center;
+}
+
+.Comment svg {
+  fill: turquoise;
+}
+.FilledLike svg {
+  fill: tomato;
+}
+
+@media (min-width: 568px) {
+  .ProfileHeader {
+    flex-direction: row;
+  }
+  .Avatar {
+    margin-left: 0%;
+    margin-right: 0%;
```

Let's add some space between the image and info
```suggestion
    margin: 0 32px 0 0;
```